client/state/selectors/is-requesting-reader-teams.js

```diff
@@ -0,0 +1,3 @@
+export default function isRequestingReaderTeams( state ) {
+	return !! state.reader.teams.isRequesting;
```

Comment:
to add some code for my earlier comment…

```js
const isRequestReaderTeams = state => null === state.reader.teams.items;
```

how can we rely on this? well, if the middleware is taking care of making sure that the data we need is loaded then _if we don't currently have the data we need_ then we are constantly in a state of _loading it_.

the question comes down to whether or we need to represent that some process is currently going on in order to update our data or whether we just need to show an initial state where no data has yet come in.

do we need to show "the team information is currently being updated" or is it enough that the data will be updated whenever a new request returns from the server?